packages/atomic/src/text/LocalizeText.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import classnames from 'classnames';
+import { WidthProperty } from 'csstype';
+
+import {
+  LocalizeProps,
+  LocalizeThemeProps,
+} from '@seolhun/localize-components-styled-types';
+
+const DEFAULT_CLASSNAME = '__Localize__Text';
+type HeadingProps = React.HTMLAttributes<HTMLHeadingElement>;
+type LocalizeTextType = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p';
+
+interface LocalizeTextProps extends LocalizeProps, HeadingProps {
+  type: LocalizeTextType;
+
+  color?: keyof LocalizeThemeProps['colors'];
+
+  weight?: WidthProperty<string>;
+}
+
+const getStyleLocalizeText = (type: LocalizeTextType) => {
+  return styled[type]<LocalizeTextProps, LocalizeThemeProps>(
+    ({ theme, color }) => {
+      const fonts = theme.fonts[type];
```

Comment:
Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/shun10114/localize-components/pullRequest?prid=5930444)